src/settings/util/convertData.js

```diff
@@ -52,7 +52,9 @@ export const filteringConditionsToFormData = (filteringConditions = [], setsFilt
 
 export const filteringConditionsToDtoFormat = (filteringConditions = []) => ({
   [SET_FIELDS.FILTERING_CONDITIONS]: filteringConditions
-    .filter(filteringCondition => filteringCondition[FILTERING_CONDITIONS_FIELDS.ACTIVE])
+    .filter((filteringCondition) => filteringCondition[FILTERING_CONDITIONS_FIELDS.ACTIVE])
+    .filter((filteringCondition) => filteringCondition[FILTERING_CONDITIONS_FIELDS.VALUE])
+    .filter((filteringCondition) => filteringCondition[FILTERING_CONDITIONS_FIELDS.SET_SPEC])
```

Comment:
I wonder whether three filter calls can be replaced with just one. Although if the dataset is small it is not a big deal.

Reply:
Fixed